### DISCUSSION TITLE
Scene follow action

### DISCUSSION THREAD

**Discussion by Zennro-hoa** [helgoboss/helgobox#1094] [Ideas]
Hi, here is the idea to go automatically from a scene to the next one when the previous has been cued. 
Best,

**Comment by helgoboss** [comment]
Hey, I just converted our previous discussion to an Idea. It contained more info. https://github.com/helgoboss/helgobox/discussions/1092

Closing this one.

**Comment by Zennro-hoa** [comment]
Thanks ! Will you include it in the next update ?

**Reply by helgoboss** [reply]
No. Until the final release, I will mainly fix bugs and improve documentation. After the final release, I will see what to implement next, starting from the most requested and/or easiest-to-implement features.

**Comment by Zennro-hoa** [comment]
Hi Benjamin,

Thanks for your quick answer and for this tool. Please consider to have it
in your list. I think it is really a plus for Live Sound configurations.
Best,
Rozenn

Le sam. 17 août 2024 à 22:49, Benjamin Klum ***@***.***> a
écrit :

> No. Until the final release, I will mainly fix bugs and improve
> documentation. After the final release, I will see what to implement next,
> starting from the most requested and/or easiest-to-implement features.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/helgoboss/helgobox/discussions/1094#discussioncomment-10368511>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPSCZYLFH3NU5NZPJGKEPDZR6ZODAVCNFSM6AAAAABMOU47JWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZWHA2TCMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Rozenn Lièvre

sie/ihr, she/her, elle
--------------------------------------------
*Sound Design*
*Sound Engineer*
--------------------------------------------
E-mail: ***@***.***
Tel (DE) : (+49) 0 176 58613088
Website: www.rozennlievre.com
---------------------------------------------
